src/operator/quantization/quantized_elemwise_add.cc

```diff
@@ -127,7 +127,6 @@ and max thresholds representing the threholds for quantizing the float32 output
 
 // TODO(zhangrong): need extra condition check if there's benefited if it's switched on
 // Since it's not compute-intensive.
-#if 0
```

Comment:
Is the `TODO` here resolved now?

Reply:
I'll remove this `todo` since this operator is registered as `kSupport` quantize mode.